jaeger-apachehttpclient/src/main/java/com/uber/jaeger/httpclient/TracingRequestInterceptor.java

```diff
@@ -77,6 +77,8 @@ public void process(HttpRequest httpRequest, HttpContext httpContext)
         Tags.PEER_PORT.set(clientSpan, (short) host.getPort());
       }
 
+      onSpanStart(clientSpan, httpRequest);
```

Comment:
I would pass httpcontext as well.  And we can have a similar beforeSpanFinished() method in the response interceptor, just to be symmetrical (in case users want to add more info like error codes or what not).